aten/src/ATen/function_wrapper.py

```diff
@@ -608,103 +632,125 @@ def to_return_type(arg, option):
     }
 
 
-def create_generic(top_env, declarations):
-    # type: (TopEnvironment, List[FunctionOption]) -> List[OutputDeclaration]
-    # translates defaults from cwrap types to C++ values
-    def translate_default(argument, type_str, default):
-        # type: (THFormal, str, Any) -> Any
-        if default is None:
-            # cause the default constructor for the object to run
-            return '{}'
-        if 'if_true' in argument:
-            return argument['default'] == argument['if_true']
-        for pattern, replacement in HEADER_CONSTANT_REPLACEMENTS:
-            default = re.sub(pattern, replacement, str(default))
-        if type_str in {'Scalar', 'int64_t', 'double'}:
+# translates defaults from cwrap types to C++ values
+def translate_default(argument, type_str, default):
```

Comment:
It looks like there's a lot of changes here, but it's really just moving `translate_default`, `translate_formal`, `get_formals`, `find_dispatch_tensor` out of `create_generic` so I can use them elsewhere.

Reply:
It'll be a lot easier to review the PR if you give us a billing of changes. Just a blow-by-blow, "Here are the logical units of change I made in this PR". Then we'll know what to expect in review.